### PR TITLE
Add support of LG Optimus Vu 

### PR DESCRIPTION
**Pull request by clamor-s**
This patch set contains next changes:
- added solomon ssd2825 spi lvds/dsi bridge and dsi panels used in p895 and p880
- fixed rtc address of pmic (needs verification that it does not brake groupers pmic)
- converted to OF backlight controller (lm3533 all cells), als/pxs sensor (adps990x), flash led (lm3560)
- converted als/pxs (adps990x) to IIO
- adjusted coupled mic-hp to properly work
- fully routed sound (max98089)
- separated tsc2007 adc part from touchscreen and added there x3 specific channel
- added 266.5 MHz peripheral opps entry
- fuel gauge driver updated to use battery cell, charger and sensors data
- added optional regulator support for ina2xx adc
- added extcon driver for max14526 muic
- added driver for max8971 charger
- added driver for gnss (it giver no errors but can not be tested)
- added P895 device tree (once P880 tree will be ready I will factor out common part like with transformers), it has excessive configuration for now because some of stuff is still in my forks under development, though this nodes should not brake anything.

### CODE REVIEW

File: drivers/gpu/drm/panel/panel-koe-tx13d100vm0eaa.c [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+	if (priv->prepared)
+		return 0;
+
+	ret = regulator_enable(priv->vcc_supply);
+	if (ret < 0) {
+		dev_err(dev, "failed to enable vcc power supply\n");
+		return ret;
+	}
+
+	mdelay(2);
```
Comment (digetx) [inline drivers/gpu/drm/panel/panel-koe-tx13d100vm0eaa.c:130]: `mdelay` almost always is a wrong choice, please see https://www.kernel.org/doc/Documentation/timers/timers-howto.txt

Comment (clamor-s) [reply drivers/gpu/drm/panel/panel-koe-tx13d100vm0eaa.c]: resolved I hope

**Comment by digetx** [comment]
Feel free to merge this once you think it's ready

**Comment by clamor-s** [comment]
@digetx may I create a new `stable` branch for pmOS based on stable linux releases?

**Comment by digetx** [comment]
Feel free to create it

**Pull request merged**